src/TableFunctions/TableFunctionS3.cpp

```diff
@@ -100,7 +101,7 @@ void TableFunctionS3::parseArgumentsImpl(ASTs & args, const ContextPtr & context
                 no_sign_request = true;
                 args_to_idx = {{"format", 2}};
             }
-            else if (second_arg == "auto" || FormatFactory::instance().getAllFormats().contains(second_arg))
+            else if (second_arg == "auto" || FormatFactory::instance().checkFormatName(second_arg))
```

Comment:
It's not correct, `checkFormatName` throws an exception, but here we need `bool` result if format exists or not